README.md

```diff
@@ -16,41 +19,46 @@ For tutorials, samples, quick starts and other documentation, visit [Azure for J
-- [Storage Blobs](sdk/storage/azure-storage-blob/README.md)
-- [Storage Blobs Batch](sdk/storage/azure-storage-blob-batch/README.md)
-- [Storage Blobs Cryptography](sdk/storage/azure-storage-blob-cryptography/README.md)
-- [Storage Queues](sdk/storage/azure-storage-queue/README.md)
+The libraries released in the GA November 2019 release:
+- [Identity](https://github.com/Azure/azure-sdk-for-java/blob/azure-identity_1.0.0/sdk/identity/azure-identity/README.md)
+- [Key Vault Keys](https://github.com/Azure/azure-sdk-for-java/blob/azure-security-keyvault-keys_4.0.0/sdk/keyvault/azure-security-keyvault-keys/README.md)
+- [Key Vault Secrets](https://github.com/Azure/azure-sdk-for-java/blob/azure-security-keyvault-secrets_4.0.0/sdk/keyvault/azure-security-keyvault-keys/README.md)
+- [Storage Blobs](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-blob_12.0.0/sdk/storage/azure-storage-blob/README.md)
+- [Storage Blobs Batch](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-blob_12.0.0/sdk/storage/azure-storage-blob-batch/README.md)
+- [Storage Blobs Cryptography](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-blob_12.0.0/sdk/storage/azure-storage-blob-cryptography/README.md)
+- [Storage Queues](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-blob_12.0.0/sdk/storage/azure-storage-queue/README.md)
 
 The libraries released in the November 2019 preview:
-- [App Configuration](sdk/appconfiguration/azure-data-appconfiguration/README.md)
-- [Event Hubs](sdk/eventhubs/azure-messaging-eventhubs/README.md)
-- [Event Hubs Checkpoint Store](sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/README.md)
-- [Storage File Shares](sdk/storage/azure-storage-file-share/README.md)
-- [Key Vault Certificates](sdk/keyvault/azure-security-keyvault-certificates/README.md)
-- [OpenCensus Tracing](sdk/core/azure-core-tracing-opencensus/README.md)
+- [App Configuration](https://github.com/Azure/azure-sdk-for-java/blob/074dd7a33f58ec02f84bfb6b94fa48529a6997ad/sdk/appconfiguration/azure-data-appconfiguration/README.md)
+- [Event Hubs](https://github.com/Azure/azure-sdk-for-java/blob/azure-messaging-eventhubs_5.0.0-preview.5/sdk/eventhubs/azure-messaging-eventhubs/README.md)
+- [Event Hubs Checkpoint Store](https://github.com/Azure/azure-sdk-for-java/blob/azure-messaging-eventhubs-checkpointstore-blob_1.0.0-preview.3/sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/README.md)
+- [Storage File Share](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-file-share_12.0.0-preview.5/sdk/storage/azure-storage-file/README.md)
+- [Key Vault Certificates](https://github.com/Azure/azure-sdk-for-java/blob/azure-keyvault-keys_4.0.0-preview.5/sdk/keyvault/azure-keyvault-certificates/README.md)
+- [OpenCensus Tracing](https://github.com/Azure/azure-sdk-for-java/tree/azure-core-tracing-opencensus_1.0.0-preview.4/sdk/tracing/azure-core-tracing-opencensus/README.md)
```

Comment:
This link is not found. Can you please check?
https://github.com/Azure/azure-sdk-for-java/tree/azure-core-tracing-opencensus_1.0.0-preview.4/sdk/tracing/azure-core-tracing-opencensus/README.md